RUNNING_THE_DEMO.md

```diff
@@ -0,0 +1,20 @@
+# Milestone 5 demo
```

Comment:
@davidmogar `README.md` is something else. WDYT of this name instead for this?

Reply:
I think we should integrate both. I like the current `README.md`, but maybe we could expand it with `## Running the demo` stuff. Maybe we could even rename the `## Tags` heading to something like `Searching for previous milestone demos?` so it's more clear.